Kitodo/src/main/webapp/WEB-INF/templates/includes/processEdit/taskList.xhtml

```diff
@@ -88,13 +92,17 @@
                 <f:param name="id" value="#{item.id}"/>
                 <i class="fa fa-pencil-square-o fa-lg"/>
             </h:link>
-            <h:commandLink id="deleteTask" title="#{msgs.loeschen}" action="#{ProzessverwaltungForm.deleteTask}"
-                           immediate="true" onclick="return confirm('#{msgs.sollDieserEintragWirklichGeloeschtWerden}')">
+            <p:commandLink id="deleteTask"
+                           title="#{msgs.loeschen}"
+                           action="#{ProzessverwaltungForm.removeTask}"
+                           immediate="true"
+                           oncomplete="jQuery('#editForm\\:saveButtonToggler').click();"
+                           update="taskTable">
```

Comment:
You remove the `onclick` confirmation for delete operation. Is this really wanted?

Reply:
> You remove the onclick confirmation for delete operation. Is this really wanted?

This was probably not intended. I re-added the confirm dialog, but used a PrimeFaces component instead of pure Javascript this time.